geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -11,7 +11,7 @@
     {% if document.has_images or document.iiif_urls %}
         {# NOTE: referencing specific version avoids an extra render-blocking lookup #}
         <script defer src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/3.0.0/openseadragon.min.js" nonce="{{ request.csp_nonce}}"></script>
```

Comment:
should we include seadragon in our npm install & bundle? (maybe to be addressed in the stimulus refactor chore)

Reply:
It makes sense for now since we just added the `"openseadragon"` webpack entry point, and I think importing there includes it in the bundle.